### ISSUE TITLE
TypeError

### ISSUE THREAD

**Issue by gartenbauer** [TarheelGrad1998/gallery-card#30]
Hi there 😊
Has this error something to do with your integration or with HACS? 

> Logger: frontend.js.latest.202112290
Source: components/system_log/__init__.py:189
First occurred: 30. Dezember 2021, 19:00:13 (91 occurrences)
Last logged: 13:00:13
http://XXX.XXX.X.X:8123/hacsfiles/gallery-card/gallery-card.js?hacstag=28428386732:177:42 Uncaught TypeError: Cannot read property 'tagName' of null

**Comment by TarheelGrad1998** [comment]
Hrm, could be me.  I might be using something not universally supported.  What browser are you using?

**Comment by gartenbauer** [comment]
I'm using:
> Microsoft Edge
Version 96.0.1054.62 (Official Build) (64-Bit)

**Comment by TarheelGrad1998** [comment]
Hrm, Edge isn't producing the error for me.  Mind sharing the yaml to your card?

**Comment by gartenbauer** [comment]
There you go:
```
type: custom:gallery-card
menu_alignment: Hidden
entities:
  - path: media-source://media_source/local/slideshow
    recursive: false
slideshow_timer: '3'
```

**Comment by TarheelGrad1998** [comment]
Thanks, for the life of me, I can't reproduce the error, but I'll take a shot at a fix anyway.

**Comment by gartenbauer** [comment]
Thank you very much 🎉
I will close this issue because the functionality is not impaired.

**Comment by TarheelGrad1998** [comment]
This is hopefully fixed in v3.3, just released.
